pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -262,11 +262,12 @@ func (c *Reconciler) reconcileRoute(ctx context.Context, service *v1alpha1.Servi
 	desiredRoute, err := resources.MakeRoute(service)
 	if err != nil {
 		// This should be unreachable as configuration creation
-		// happens first in reconcile()
+		// happens first in `reconcile()` and it verifies the edgecases
+		// that would make `MakeRoute` fail as well.
```

Comment:
Can you duplicate this update to line 254 as well? Everything else looks good.

Thanks for the clean up!

Reply:
Done.